src/main/webapp/js/instructorCourseEdit.js

```diff
@@ -70,6 +72,11 @@ function disableFormEditInstructor(number) {
 function showNewInstructorForm() {
     $('#panelAddInstructor').show();
     $('#btnShowNewInstructorForm').hide();
+    if (hasClickedAddInstructorBefore === false) {
```

Comment:
I feel that it might be better to simply bind it upon `document.ready` rather than having to go through this check when adding a new instructor